ReactCommon/react/renderer/components/view/conversions.h

```diff
@@ -382,7 +383,8 @@ inline void fromRawValue(
   } else if (value.hasType<std::string>()) {
     const auto stringValue = (std::string)value;
     if (stringValue == "auto") {
-      result = YGValueUndefined;
+      auto reactNativeConfig = context.contextContainer.at<std::shared_ptr<ReactNativeConfig const>>("ReactNativeConfig");
```

Comment:
The conversion code here is going to be a relatively hot path, so it might make sense to cache this (e.g. in a static variable), since I think the config values shouldn't change over app lifetime. Can make that change when importing, but wanted to give a quick heads up.